Cargo.toml

```
@@ -44,6 +44,7 @@ serde = { version = "1", features = ["derive"] }
 serde_json = "1"
 serde_yaml = "0.9"
 spreet = { version = "0.8", default-features = false }
+sqlite-hashes = { version = "0.1", features = ["sha256"] }
```

By default, it already includes sha256 - see [here](https://github.com/nyurik/sqlite-hashes/blob/0095dd843aa08906ff822b387014375bae5a6bef/Cargo.toml#L14)

```suggestion
sqlite-hashes = "0.1"
```